Documentation/Cluster.rst

```diff
@@ -47,11 +47,15 @@ Joining the node to the cluster
 
 **Note:** The following can only be performed by a superuser.
 
+**Note:** Both nodes must allow root login over SSH.

```

Comment:
Possibly mention that it is only required from the network chosen for MCVirt clustering